.github/workflows/e2e.yml

```
@@ -158,7 +158,7 @@ jobs:
       - name: e2e/docker-login
         uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
         # Do not authenticate on Forks
-        if: github.event.pull_request.head.repo.full_name == github.repository
+        if: "! github.event.pull_request.head.repo.fork "
```

Good find Claudio, thanks! This requires a bit more tweaking, because on `main` branch pushes the `github.event.pull_request` field will be empty.
```suggestion
        if: "github.event.repository.full_name == github.repository"
```

---

Thanks!